### PR TITLE
SISRP-25805 - Front-End Development for Student Overview (Advisor) - Student Success - Units & Finances Sections

### PR DESCRIPTION
**Pull request by davidlimcheng**
https://jira.berkeley.edu/browse/SISRP-25805

* Surfaces Units & Finances onto Student Success
* I don't think the Design JIRA has been updated to the most recent design, seen here:
studentsuccessmvp
* Left out the "click-to-expand" feature, due to it being inconsistent with the GPA section.
* This should be my last PR for the day/7.1

### CODE REVIEW

File: src/assets/templates/widgets/student_success/finances.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<div class="cc-student-success-subtitle">
+    <h3>
+      Finances
+    </h3>
```
Comment (johncrossman) [inline src/assets/templates/widgets/student_success/finances.html]: Minor indentation fix needed, @davidlimcheng

File: src/assets/templates/widgets/student_success/units.html [hunk 2]
```diff
@@ -0,0 +1,38 @@
+<div class="cc-student-success-subtitle">
+    <h3>
+      Units
+      <p data-ng-if="!gpaUnits.isLegacy">(Cumulative <span data-ng-bind="gpaUnits.totalUnits | number:1"></span>)</p>
+    </h3>
+</div>
```
Comment (johncrossman) [inline src/assets/templates/widgets/student_success/units.html]: @davidlimcheng ditto on indentation

**Comment by davidlimcheng** [comment]
Fixed, thanks @johncrossman.

File: src/assets/templates/widgets/student_success/units.html [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+<div class="cc-table cc-student-success-term-gpa">
+  <table>
+    <tbody data-ng-if="!gpaUnits.isLegacy">
```
Comment (johncrossman) [inline src/assets/templates/widgets/student_success/units.html]: @davidlimcheng from what I see in the `MyAcademics::GpaUnits` class, the value of `isLegacy` is always true. I'm probably missing something. Can you explain.

Comment (raydavis) [inline src/assets/templates/widgets/student_success/units.html]: @johncrossman , in https://github.com/ets-berkeley-edu/calcentral/blob/master/app/models/my_academics/gpa_units.rb#L8 , ``oracle_gpa_units`` is not looked at when CS/Hub is being used. ``isLegacy`` is only set by the Oracle method.

File: src/assets/templates/widgets/student_success/finances.html [hunk 4]
```diff
@@ -0,0 +1,17 @@
+<div class="cc-student-success-subtitle">
+  <h3>
+    Finances
+  </h3>
+</div>
+<div class="cc-table cc-student-success-term-gpa">
```
Comment (johncrossman) [inline src/assets/templates/widgets/student_success/finances.html]: @davidlimcheng not a big deal, but you're using `cc-student-success-term-gpa` in the Finances sub-widget. Can we rename the style (in all usages) to the more generic `cc-student-success-sub-section`?  (Make the change in a future PR if you'd like.)

File: src/assets/templates/widgets/student_success/units.html (hunk 3 above)
Comment (johncrossman) [inline src/assets/templates/widgets/student_success/units.html]: Ah, got it. Thanks @raydavis

File: src/assets/templates/widgets/student_success/finances.html (hunk 4 above)
Comment (davidlimcheng) [inline src/assets/templates/widgets/student_success/finances.html]: @johncrossman Ahh that is very sloppy indeed - I'll amend this PR.  Great catch.

Comment (davidlimcheng) [inline src/assets/templates/widgets/student_success/finances.html]: Amended

**Comment by johncrossman** [comment]
Thanks @davidlimcheng   Waiting on Mr. T

**Pull request merged**